src/main/kotlin/com/openlattice/chronicle/services/timeusediary/TimeUseDiaryService.kt

```diff
@@ -195,21 +204,29 @@ class TimeUseDiaryService(
 
     private fun getTimeUseDiarySummarizedData(
         studyId: UUID,
-        startDate: OffsetDateTime,
-        endDate: OffsetDateTime
+        participantIds: Set<String>?,
+        startDate: OffsetDateTime?,
+        endDate: OffsetDateTime?
     ): Iterable<List<Map<String, Any>>> {
         try {
             val hds = storageResolver.getPlatformStorage()
             val iterable = BasePostgresIterable(
                 PreparedStatementHolderSupplier(
                     hds,
-                    downloadSummarizedTimeUseDiaryDataSql,
+                    getSummarizedTimeUseDiarySql(participantIds, startDate, endDate),
                     1024
                 ) { ps ->
                     var index = 0
                     ps.setObject(++index, studyId)
-                    ps.setObject(++index,startDate)
-                    ps.setObject(++index, endDate)
+                    participantIds?.let {
+                        ps.setArray(++index, PostgresArrays.createTextArray(hds.connection, it))
+                    }
+                    startDate?.let {
```

Comment:
Same comment as before, try to minimize the number of different queries that have to be tested (it would be different if startDate/endDate were different columns that weren't included in the return data as this would save the  columnar db from reading that column when processing the data.)